kubectl-plugin/pkg/cmd/session/session.go

```diff
@@ -102,11 +112,22 @@ func NewSessionCommand(cmdFactory cmdutil.Factory, streams genericiooptions.IOSt
 	}
 
 	cmd.Flags().BoolVarP(&options.Verbose, "verbose", "v", false, "verbose output")
-
+	cmd.Flags().BoolVarP(&options.killAll, "kill-all", "", false, "kill all existing Ray sessions started by kubectl-ray session command")
 	return cmd
 }
 
 func (options *SessionOptions) Complete(cmd *cobra.Command, args []string) error {
+	if options.killAll {
+		if len(args) > 0 {
+			return cmdutil.UsageErrorf(cmd, "accepts no args when killAll flag is set")
+		}
+		return nil
+	}
+
+	if len(args) == 0 {
+		return cmdutil.UsageErrorf(cmd, "killAll flag is not set, but no args provided")
+	}
+
```

Comment:
Create another function called `Validate` and put the validation log there. You can use other files for reference. The order is `Complete`, `Validate`, `Run`.